src/tasks/installers/operator.ts

```diff
@@ -376,15 +382,6 @@ export class OperatorTasks {
         }
       }
     },
-    {
```

Comment:
I understand that we should not delete the CRD when it might still be used by other installations.
OTOH for now, on the `chectl server:start`, the CRD is only created if it didn't exist as shown [here](https://github.com/che-incubator/chectl/blob/e1af325ea7ab114a7a6d04c423b3cf2a2a99cf93/src/tasks/installers/operator.ts#L134).
That means that deleting a Che server with a `chectl` version and reinstalling with a new version of `chectl` will not update the CRD anymore.
That's a hard-to-solve question here, since we can manage only one version of the CRD.
That's a typical case where OLM-based installation has a real value, because the logic of CRD update is managed by OLM itself.

Reply:
cc @l0rd

Reply:
@davidfestal 
If we are not going to delete CRD file, I would suggest to update it every time we install che using checlt

Reply:
It allows to keep CRD up to date